ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
```

Comment:
Please add this to a good place in https://github.com/ibmdbanalytics/ibmdbpy/blob/master/docs/source/frame.rst

Reply:
I have added delete_na to the filtering section of frame.rst